Rings/TargetCasting.lua

```diff
@@ -134,7 +134,7 @@ function module:OnModuleEnable()
 		self:RegisterUnitEvent("UNIT_SPELLCAST_DELAYED")
 		self:RegisterUnitEvent("UNIT_SPELLCAST_CHANNEL_START")
 		self:RegisterUnitEvent("UNIT_SPELLCAST_CHANNEL_UPDATE")
-		if not (ArcHUD.isClassicWoW or ArcHUD.isClassicTbc) then
+		if not (ArcHUD.classic) then
 			self:RegisterUnitEvent("UNIT_SPELLCAST_INTERRUPTIBLE")

```

Comment:
According to docs, this was added in 3.2, but it gives me a LUA error in game so I guess it's not in yet. I have a version of this ring for when this is enabled too.